beesdoo_product/models/beesdoo_product.py

```diff
@@ -252,12 +265,17 @@ def _compute_cost(self):
         for product in self:
             suppliers = product._get_main_supplier_info()
             if len(suppliers) > 0:
-                product.suggested_price = (
-                    suppliers[0].price * product.uom_po_id.factor
-                ) * (
-                    1
-                    + suppliers[0].product_tmpl_id.categ_id.profit_margin / 100
+                price = suppliers[0].price
+                profit_margin_supplier = suppliers[0].name.profit_margin
+                profit_margin_product_category = suppliers[
+                    0
+                ].product_tmpl_id.categ_id.profit_margin
+                profit_margin = (
+                    profit_margin_supplier or profit_margin_product_category
                 )
+                product.suggested_price = (
+                    price * product.uom_po_id.factor
+                ) * (1 + profit_margin / 100)
```

Comment:
I know it was there but why 1 + profit_margin. having 20 of profit margin make it be of 21. So I don't get it...

from my understanding it should be 1 * profit_margin...

Reply:
Order of computations: first the devision, then the addition!
`profit_margin` = 20 => `(1 + profit_margin / 100)` = 1 + 20/100 = 1.20 ✔️

Reply:
you're right !